### DISCUSSION TITLE
MQTT Data Tables Missing In Postgres

### DISCUSSION THREAD

**Discussion by syedzahidulla** [processone/ejabberd#4062] [Questions] [answered]
MQTT_SUB AND MQTT_SESSION Both these tables are visible in webadmin and its never seen in postgre still i have given db_type as sql in mod_mqtt and mqtt_pub table is present in postgre schema but no data is getting stored there. any idea what to configure more

**Comment by mremond** [comment]
Did you configure the MQTT modules to use a relational database backend and not the default mnesia backend ?

**Comment by badlop** [comment, marked as answer]
Right, there are some tables that contain runtime information, only relevant while the server is running, and its content is irrelevant once the server stops. Such tables are not stored in SQL or in disk Mnesia.

Those tables are kept in ETS or in RAM mnesia, for example: bosh, muc_online_room, route, s2s, session, session_counter, mqtt_session and mqtt_sub.
